diff -ur ../pdftk-1.41.orig/java_libs/com/lowagie/bc/asn1/Makefile ./java_libs/com/lowagie/bc/asn1/Makefile
--- ../pdftk-1.41.orig/java_libs/com/lowagie/bc/asn1/Makefile	2007-04-11 16:26:56.000000000 -0400
+++ ./java_libs/com/lowagie/bc/asn1/Makefile	2007-04-11 16:29:32.000000000 -0400
@@ -21,12 +21,13 @@
 %.class : %.java
 	$(GCJ) $(GCJFLAGS) -C $<
 
+.PRECIOUS: %.class
+
 # gcjh doesn't want the ".class" extension, so use
 # the "$*" automatic variable, here
 #
 %.h : %.class
-	$(GCJH) --classpath="." $*;
-	$(RM) $<
+	cd ../../../..; $(GCJH) --classpath="." com/lowagie/bc/asn1/$*
 
 ##
 # targets
diff -ur ../pdftk-1.41.orig/java_libs/com/lowagie/text/Makefile ./java_libs/com/lowagie/text/Makefile
--- ../pdftk-1.41.orig/java_libs/com/lowagie/text/Makefile	2007-04-11 16:26:55.000000000 -0400
+++ ./java_libs/com/lowagie/text/Makefile	2007-04-11 16:29:44.000000000 -0400
@@ -21,12 +21,13 @@
 %.class : %.java
 	$(GCJ) $(GCJFLAGS) -C $<
 
+.PRECIOUS: %.class
+
 # gcjh doesn't want the ".class" extension, so use
 # the "$*" automatic variable, here
 #
 %.h : %.class
-	$(GCJH) --classpath="." $*;
-	$(RM) $<
+	cd ../../..; $(GCJH) --classpath="." com/lowagie/text/$*
 
 ##
 # targets
diff -ur ../pdftk-1.41.orig/java_libs/com/lowagie/text/markup/Makefile ./java_libs/com/lowagie/text/markup/Makefile
--- ../pdftk-1.41.orig/java_libs/com/lowagie/text/markup/Makefile	2007-04-11 16:26:56.000000000 -0400
+++ ./java_libs/com/lowagie/text/markup/Makefile	2007-04-11 16:29:55.000000000 -0400
@@ -21,12 +21,13 @@
 %.class : %.java
 	$(GCJ) $(GCJFLAGS) -C $<
 
+.PRECIOUS: %.class
+
 # gcjh doesn't want the ".class" extension, so use
 # the "$*" automatic variable, here
 #
 %.h : %.class
-	$(GCJH) --classpath="." $*;
-	$(RM) $<
+	cd ../../../..; $(GCJH) --classpath="." com/lowagie/text/markup/$*
 
 ##
 # targets
diff -ur ../pdftk-1.41.orig/java_libs/com/lowagie/text/pdf/codec/Makefile ./java_libs/com/lowagie/text/pdf/codec/Makefile
--- ../pdftk-1.41.orig/java_libs/com/lowagie/text/pdf/codec/Makefile	2007-04-11 16:26:56.000000000 -0400
+++ ./java_libs/com/lowagie/text/pdf/codec/Makefile	2007-04-11 16:30:16.000000000 -0400
@@ -21,12 +21,13 @@
 %.class : %.java
 	$(GCJ) $(GCJFLAGS) -C $<
 
+.PRECIOUS: %.class
+
 # gcjh doesn't want the ".class" extension, so use
 # the "$*" automatic variable, here
 #
 %.h : %.class
-	$(GCJH) --classpath="." $*;
-	$(RM) $<
+	cd ../../../../..; $(GCJH) --classpath="." com/lowagie/text/pdf/codec/$*
 
 ##
 # targets
diff -ur ../pdftk-1.41.orig/java_libs/com/lowagie/text/pdf/codec/postscript/Makefile ./java_libs/com/lowagie/text/pdf/codec/postscript/Makefile
--- ../pdftk-1.41.orig/java_libs/com/lowagie/text/pdf/codec/postscript/Makefile	2007-04-11 16:26:56.000000000 -0400
+++ ./java_libs/com/lowagie/text/pdf/codec/postscript/Makefile	2007-04-11 16:30:34.000000000 -0400
@@ -21,12 +21,13 @@
 %.class : %.java
 	$(GCJ) $(GCJFLAGS) -C $<
 
+.PRECIOUS: %.class
+
 # gcjh doesn't want the ".class" extension, so use
 # the "$*" automatic variable, here
 #
 %.h : %.class
-	$(GCJH) --classpath="." $*;
-	$(RM) $<
+	cd ../../../../../..; $(GCJH) --classpath="." com/lowagie/text/pdf/codec/postscript/$*
 
 ##
 # targets
diff -ur ../pdftk-1.41.orig/java_libs/com/lowagie/text/pdf/codec/wmf/Makefile ./java_libs/com/lowagie/text/pdf/codec/wmf/Makefile
--- ../pdftk-1.41.orig/java_libs/com/lowagie/text/pdf/codec/wmf/Makefile	2007-04-11 16:26:56.000000000 -0400
+++ ./java_libs/com/lowagie/text/pdf/codec/wmf/Makefile	2007-04-11 16:30:41.000000000 -0400
@@ -21,12 +21,13 @@
 %.class : %.java
 	$(GCJ) $(GCJFLAGS) -C $<
 
+.PRECIOUS: %.class
+
 # gcjh doesn't want the ".class" extension, so use
 # the "$*" automatic variable, here
 #
 %.h : %.class
-	$(GCJH) --classpath="." $*;
-	$(RM) $<
+	cd ../../../../../..; $(GCJH) --classpath="." com/lowagie/text/pdf/codec/wmf/$*
 
 ##
 # targets
diff -ur ../pdftk-1.41.orig/java_libs/com/lowagie/text/pdf/fonts/Makefile ./java_libs/com/lowagie/text/pdf/fonts/Makefile
--- ../pdftk-1.41.orig/java_libs/com/lowagie/text/pdf/fonts/Makefile	2007-04-11 16:26:56.000000000 -0400
+++ ./java_libs/com/lowagie/text/pdf/fonts/Makefile	2007-04-11 16:30:52.000000000 -0400
@@ -30,12 +30,13 @@
 %.o : %.afm
 	$(GCJ) -c --resource com/lowagie/text/pdf/fonts/$< $< -o $@
 
+.PRECIOUS: %.class
+
 # gcjh doesn't want the ".class" extension, so use
 # the "$*" automatic variable, here
 #
 %.h : %.class
-	$(GCJH) --classpath="." $*;
-	$(RM) $<
+	cd ../../../../..; $(GCJH) --classpath="." com/lowagie/text/pdf/fonts/$*
 
 ##
 # targets
diff -ur ../pdftk-1.41.orig/java_libs/com/lowagie/text/pdf/Makefile ./java_libs/com/lowagie/text/pdf/Makefile
--- ../pdftk-1.41.orig/java_libs/com/lowagie/text/pdf/Makefile	2007-04-11 16:26:55.000000000 -0400
+++ ./java_libs/com/lowagie/text/pdf/Makefile	2007-04-11 16:30:59.000000000 -0400
@@ -21,12 +21,13 @@
 %.class : %.java
 	$(GCJ) $(GCJFLAGS) -C $<
 
+.PRECIOUS: %.class
+
 # gcjh doesn't want the ".class" extension, so use
 # the "$*" automatic variable, here
 #
 %.h : %.class
-	$(GCJH) --classpath="." $*;
-	$(RM) $<
+	cd ../../../..; $(GCJH) --classpath="." com/lowagie/text/pdf/$*
 
 ##
 # targets
diff -ur ../pdftk-1.41.orig/java_libs/com/lowagie/text/xml/xmp/Makefile ./java_libs/com/lowagie/text/xml/xmp/Makefile
--- ../pdftk-1.41.orig/java_libs/com/lowagie/text/xml/xmp/Makefile	2007-04-11 16:26:55.000000000 -0400
+++ ./java_libs/com/lowagie/text/xml/xmp/Makefile	2007-04-11 16:31:08.000000000 -0400
@@ -21,12 +21,13 @@
 %.class : %.java
 	$(GCJ) $(GCJFLAGS) -C $<
 
+.PRECIOUS: %.class
+
 # gcjh doesn't want the ".class" extension, so use
 # the "$*" automatic variable, here
 #
 %.h : %.class
-	$(GCJH) --classpath="." $*;
-	$(RM) $<
+	cd ../../../../..; $(GCJH) --classpath="." com/lowagie/text/xml/xmp/$*
 
 ##
 # targets
