--- pdftk-1.41/pdftk/Makefile.RedHat.org	2006-09-21 19:37:38.000000000 +0200
+++ pdftk-1.41/pdftk/Makefile.RedHat	2007-02-26 20:50:28.000000000 +0100
@@ -26,12 +26,12 @@
 # if you want pdftk to ask before overwriting a file, set
 # ASK_ABOUT_WARNINGS to true; otherwise: false; override this default
 # with the dont_ask or do_ask command-line options
-CPPFLAGS= -O2 -DPATH_DELIM=0x2f -DASK_ABOUT_WARNINGS=false -fdollars-in-identifiers
+CPPFLAGS= ${RPM_OPT_FLAGS} -DPATH_DELIM=0x2f -DASK_ABOUT_WARNINGS=false -fdollars-in-identifiers
 CXXFLAGS= -lgcj
 
 # itext compiler flags
 # -O3 might cause pdftk to segfault on cat operation (gcc 3.4.4)
-export GCJFLAGS= -O2
+export GCJFLAGS= ${RPM_OPT_FLAGS}
 
 #
 export ARFLAGS= rs
